interface/Cargo.toml

```
@@ -0,0 +1,59 @@
+[package]
+name = "solana-stake-interface"
+version = "0.1.0"
+description = "Instructions and constructors for the Stake program"
+repository = "https://github.com/solana-program/stake"
+edition = "2021"
+readme = "README.md"
+license-file = "../LICENSE"
+
+[dependencies]
+borsh = { version = "1.5.1", features = ["derive", "unstable__schema"], optional = true }
+borsh0-10 = { package = "borsh", version = "0.10.3", optional = true }
+num-traits = "0.2"
+serde = { version = "1.0.210", features = ["derive"], optional = true }
+serde_derive = { version = "1.0.210", optional = true }
+solana-decode-error = "^2.1"
+solana-clock = "^2.1"
+solana-cpi = { version = "^2.1", optional = true }
+solana-frozen-abi = { version = "^2.1", features = ["frozen-abi"], optional = true }
+solana-frozen-abi-macro = { version = "^2.1", features = ["frozen-abi"], optional = true }
+solana-instruction = "^2.1"
+solana-program-error = { version = "^2.1", optional = true }
+solana-pubkey = { version = "^2.1", default-features = false }
+solana-system-interface = { version = "0.0.1", git = "https://github.com/solana-program/system.git" }
+
+[dev-dependencies]
+assert_matches = "1.5.0"
+bincode = "1.3.3"
+solana-borsh = "^2.1"
+solana-program = { version = "^2.1", default-features = false }
+static_assertions = "1.1.0"
+strum = "0.24"
+strum_macros = "0.24"
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
```

Wanna add the extra docs bits here?
```suggestion
[package.metadata.docs.rs]
targets = ["x86_64-unknown-linux-gnu"]
all-features = true
rustdoc-args = ["--cfg=docsrs"]
```